source/code/plugins/out_oms_blob.rb

```diff
@@ -227,9 +233,9 @@ def handle_records(tag, records)
       @log.debug "Success getting the BLOB uri in #{time.round(3)}s"
 
       start = Time.now
-      append_blob(blob_uri, records)
+      dataSize = append_blob(blob_uri, records)
       time = Time.now - start
-      @log.debug "Success sending the data to BLOB #{time.round(3)}s"
+      @log.debug "Success sending #{dataSize} bytes data to BLOB #{time.round(3)}s"
```

Comment:
Success sending #{dataSize} bytes **of** data ...

Insert "of" for grammatical correctness